src/iterative/KOSMAPOSL/KOSMAPOSLReconstruction.cxx

```diff
@@ -302,6 +258,51 @@ set_up(shared_ptr <TargetT > const& target_image_ptr)
  if (base_type::set_up(target_image_ptr) == Succeeded::no)
      error("KOSMAPOSL::set_up(): Error building post filter");
```

Comment:
actually, lots of other things could have gone wrong. let's just say
```suggestion
     error("KOSMAPOSL::set_up(): Error setting-up underlying OSMAPOSLReconstruction object");
```
or similar